src/Uno.UI/UI/Xaml/Controls/SplitView/SplitView.cs

```diff
@@ -384,6 +384,9 @@ private void UpdateVisualStates(bool useTransitons)
 				PaneOpening?.Invoke(this, null);
 			}
 
+#if __IOS__
+			PatchInvalidFinalState(stateName);
```

Comment:
Create a partial method and you won't need the `#if __IOS__` :-).

Reply:
but it's really a patch which should never be implemented on other platforms. Generally I'd agree with partial method but in this case it might be better to keep the #if __ IOS __ ? I'm ok with either.

Reply:
Indeed I asked myself if I should create a partial method for that or not, but I think that partial methods should be see as "optional feature", like `OnLoadedPartial` that you **can** implement to do something platform specific. 
Here we **must** invoke this method on iOS, and it's not something that you may implement if needed on another platform.